tests/test_config.py

```diff
@@ -60,6 +60,19 @@ def test_transparent(config):
     np.testing.assert_allclose(frame[0, 0], [0, 0, 0, 0])
 
 
+def test_transparent_by_background_opacity(config):
```

Comment:
```suggestion
def test_transparent_by_background_opacity(config, dry_run):
```
and then remove the `config.dry_run = True` (this was introduced in #3853 )

Reply:
Done: 38ec8ee5